### DISCUSSION TITLE
I want to add the 'vue3-popper' module to 'nuxt.config.ts'.

### DISCUSSION THREAD

**Discussion by flaghrbs1004** [nuxt/nuxt#24739] [Questions] [answered]
First, the reason why we want to include the 'vue3-popper' module is because [bootstrap5 dropdowns](https://getbootstrap.kr/docs/5.0/components/dropdowns/#usage) does not function.
After Googling, I came up with the hypothesis that 'popper' was not installed properly.
![image](https://github.com/nuxt/nuxt/assets/8098013/db9826a8-3489-4199-8eb9-f7672e87fc8a)

So I thought I should install ‘vue3-popper’.
![image](https://github.com/nuxt/nuxt/assets/8098013/0ca8d53a-04cf-4124-9601-3b38d6d21320)

But it is showing me this error.
![image](https://github.com/nuxt/nuxt/assets/8098013/45c38fdd-157d-434c-ba97-4ac3fda2b2a9)

I also tried adding '@'.
![image](https://github.com/nuxt/nuxt/assets/8098013/82c28557-74b1-4d39-9676-71dfe53ace8f)

Maybe I didn't import bootstrap5 properly?

Or, please let me know if the vue3-popper import method is wrong. Thank you.

And I have one more question.

In my nuxt.config.ts there is a red line in css. The CSS is imported normally, but I'm concerned about the red line.
![image](https://github.com/nuxt/nuxt/assets/8098013/a7bd1e44-4506-4f7c-9945-c5f9867653b9)

**Comment by masyoudi** [comment, marked as answer]
vue3-popper is not provide nuxt modules, if you need vue3-popper globally you should create plugins like this

`/plugins/vue3-poppover.js`
```js
import Popper from 'vue3-popper';

import { defineNuxtPlugin } from '#app';

export default defineNuxtPlugin(({ vueApp }) => {
  vueApp.component('Popper', Popper);
});
```
